src/modules/compliance/src/lib/procedures/testingProcedures.schema.json

```
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "audit": {
+      "anyOf": [
+        {
+          "type": "object",
+          "required": [
+            "auditFailure"
+          ],
+          "properties": {
+            "auditFailure": {
+              "type": "object",
+              "additionalProperties": false
```

Could you add a message property:
```suggestion
              "additionalProperties": false,
              "properties": {
                "message": {
                  "type": "string"
                }
              },
```